src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer() {
+    }
```

Comment:
There is no need in default constructors here, they are available by default if there are no other constructors defined.
Same for all classes